test/lexical/collections/Upload/index.ts

```diff
@@ -34,4 +34,19 @@ const Uploads: CollectionConfig = {
   },
 }
 
-export default Uploads
+export const Uploads2: CollectionConfig = {
+  ...Uploads,
+  slug: uploads2Slug,
+}
+
+export const Uploads3: CollectionConfig = {
+  ...Uploads,
+  slug: uploads3Slug,
+  fields: [
+    ...Uploads.fields,
+    {
+      name: 'altText',
+      type: 'text',
+    },
+  ],
+}
```

Comment:
Nit: Can we use 2 instead of 3 collections?

Reply:
done!